### PR TITLE
Created ConditionThumbnail component, added to ConditionsList

### PR DESCRIPTION
**Pull request by emlabelle**
Should map over the conditions object.

## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/CONTRIBUTING.md)
- [ ] Double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

File: app/components/ConditionThumbnail/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  background: #eee;
+  border: 1px black solid;
+  margin: 0 auto;
+  padding: 5px;
+`;
+
+function ConditionThumbnail(props) {
+  return (
+    <Wrapper>
+      <img src="https://placebear.com/100/100" alt="" />
```
Comment (jenniferwiner) [inline app/components/ConditionThumbnail/index.js]: Wow https://placebear.com/ is awesome!

File: app/containers/ConditionsList/index.js [hunk 2]
```diff
@@ -33,6 +34,18 @@ export class ConditionsList extends React.Component {
           <meta name="description" content="Description of ConditionsList" />
         </Helmet>
         <h1>Conditions List</h1>
+        {!conditions.length ? (
+          <h2>No conditions found</h2>
+        ) : (
+          conditions.map(condition => (
+            <ConditionThumbnail
+              key={condition.date}
```
Comment (jenniferwiner) [inline app/containers/ConditionsList/index.js]: @emlabelle - is date unique? could we have any issues using it as the key

**Comment by jenniferwiner** [comment]
Thanks for the PR @emlabelle - just one question and then i'm good to merge. hope to see you at the meetup on Thursday!

File: app/containers/ConditionsList/index.js (hunk 2 above)
Comment (emlabelle) [reply app/containers/ConditionsList/index.js]: if it uses DateTime.now it should work on a small scale. For more users we would need a better solution. I found some suggestions here for making the date object unique: https://stackoverflow.com/questions/5608980/how-to-ensure-a-timestamp-is-always-unique

Comment (jenniferwiner) [reply app/containers/ConditionsList/index.js]: Gotcha - We may think about this for a future fix but I am good to merge it in. Thanks for contributing!

**Pull request merged**